### PR TITLE
testcase/drivers: fix drivers tc

### PR DESCRIPTION
**Pull request by deepaksrma**

### CODE REVIEW

File: apps/examples/testcase/le_tc/drivers/tc_rtc.c [hunk 1]
```diff
@@ -104,7 +104,7 @@ static void tc_driver_rtc_ioctl(void)
 	rtctime_s.tm_sec = 3;
 	rtctime_s.tm_mday = 1;
 	rtctime_s.tm_mon = 2;
-	rtctime_s.tm_year = 3;
+	rtctime_s.tm_year = 71;
```
Comment (amogh-samsung) [inline apps/examples/testcase/le_tc/drivers/tc_rtc.c:107]: LGTM

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Comment by sunghan-chang** [comment]
@deepaksrma Could you let me know test results on local time enabled and disabled both?

**Comment by deepaksrma** [comment]
> @deepaksrma Could you let me know test results on local time enabled and disabled both?

results are same with enabling localtime or disabling it. 
tested only rtc drivers TCs.

```
TASH>>drivers_tc
TASH>>
########## Drivers TC Start ##########

[tc_driver_rtc_open_close] PASS

[tc_driver_rtc_read_write] PASS
[tc_driver_rtc_ioctl] PASS

########## Drivers TC End [PASS : 3, FAIL : 0] ##########
```

**Comment by thapav** [comment]
May be you could add the test results in the commit description for respective tc's

**Comment by deepaksrma** [comment]
> May be you could add the test results in the commit description for respective tc's

I don't think this is necessary because I have already mentioned that only tc_driver_rtc_ioctl is failing and with this patch it is passed.

**Review by an4967 — APPROVED** [review]
LGTM

**Review by jsdosa — APPROVED** [review]
LGTM.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**